tests/zfs-tests/tests/perf/perf.shlib

```diff
@@ -233,17 +233,17 @@ function do_collect_scripts
 	typeset oIFS=$IFS
 	IFS=','
 	for item in $PERF_COLLECT_SCRIPTS; do
-		collect_scripts+=($(echo $item | sed 's/^ *//g'))
+		collect_scripts+=($(echo "$item" | sed 's/^ *//g'))
 	done
 	IFS=$oIFS
 
 	typeset idx=0
 	while [[ $idx -lt "${#collect_scripts[@]}" ]]; do
 		typeset logbase="$(get_perf_output_dir)/$(basename \
-		    $SUDO_COMMAND)"
+		    "$SUDO_COMMAND")"
```

Comment:
Again quotes inside quotes.

Reply:
Please refer to reply about subshell above.